src/Symfony/Bundle/FrameworkBundle/Resources/config/validator.xml

```diff
@@ -47,7 +48,6 @@
                     </call>
                 </service>
             </argument>
-            <deprecated>The "%service_id%" service is deprecated since Symfony 2.8 and will be removed in 3.0.</deprecated>
```

Comment:
We can't remove it in 3.0 now, so let's revisit this later in 3.0 (see if we want to deprecate it there).